src/server/pps/server/master.go

```diff
@@ -479,53 +479,40 @@ func (a *apiServer) monitorPipeline(pachClient *client.APIClient, pipelineInfo *
 		})
 		eg.Go(func() error {
 			return backoff.RetryNotify(func() error {
-				if err := setPipelineState(pps.PipelineState_PIPELINE_STANDBY); err != nil {
-					return err
-				}
-				standby := true
+				standby := false
 				for {
```

Comment:
So this would be good to confirm experimentally but I believe the issue with this code is that while the pipeline is in standby this for loop will busy wait. Because the select has a `default` it will always enter that block when there's not a new commit available, notice that there's nothing to do because it's already in standby, loop and repeat that process. I suspect if you run this and put a pipeline in standby you'll see a pachd node that should be idling consuming 100% of a cpu core. Assuming this is the case this necessitates 2 separate selects because there's no way to have a conditional `default` block and we only want the `default` block to be active if we're not already in stand by.